packages/components/package-lock.json

```diff
@@ -115,39 +115,6 @@
 				}
 			}
 		},
-		"@jobber/design": {
```

Comment:
I'm not so sure about some of these changes, not what I was expecting or intending to do here

I removed rrule and did `npm ci`

Reply:
yeah, that's some lerna magic that I still don't understand. Basically running `npm install` on the components folder yields a different result than doing it on the root folder. It all boils down to the `lerna bootstrap` doing it's magic on each package to link them up.

one day, we'll figure out what it's doing. but that day is not today haha. It also has a specific way of adding a package (that I just learned a week ago)

```
npx lerna add rrule@latest packages/components
```

which runs `npm install` on packages/components but with more magic like not doing this unintended change. probably.

Reply:
seems like this was incorrectly added, and now it is being removed as it should be

the git history says I added jobber/design to the /components level, which isn't what we want I'm pretty sure